module-presentation/src/main/kotlin/site/yourevents/health/api/HealthApi.kt

```diff
@@ -0,0 +1,16 @@
+package site.yourevents.health.api
+
+import DisableSwaggerSecurity
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.web.bind.annotation.GetMapping
+import site.yourevents.response.ApiResponse
+
+@Tag(name = "상태 확인")
+interface HealthApi {
+
+    @DisableSwaggerSecurity
+    @GetMapping("/health-check")
+    @Operation(summary = "서버 상태 확인")
+    fun healthCheck(): ApiResponse<String>
```

Comment:
health-check의 경우, 굳이 불필요한 반환값을 위해 body에 String 형식의 자료형을 담아줄 필요는 없을 것 같아보여요. 그 대신 자바의 `Void`형을 대체하는 `Unit`을 통해 성공 코드만 반환하는 것은 어떻게 생각하시나요??

```suggestion
    fun healthCheck(): ApiResponse<Unit>
```

Reply:
좋은 방법이라고 생각합니다. 수정하도록 하겠습니다!